stdlib/LibUV_jll/src/LibUV_jll.jl

```diff
@@ -23,7 +23,7 @@ if Sys.iswindows()
 elseif Sys.isapple()
     const libuv = "@rpath/libuv.2.dylib"
 else
-    const libuv = "libuv.so.2"
+    const libuv = "libuv.so"
 end
```

Comment:
I'm not sure why, but on Windows only libuv-2.dll is in the ZIP, not libuv.dll like for other libraries. So I skipped it on Windows and Mac to avoid breaking something (I couldn't check Mac).